packages/vector_graphics_compiler/test/parsers_test.dart

```diff
@@ -172,6 +172,10 @@ void main() {
     expect(parseDoubleWithUnits('1pt', theme: const SvgTheme()), 1 + 1 / 3);
   });
 
+  test('"none" conversion', () {
+    expect(parseDoubleWithUnits('none', theme: const SvgTheme()), null);
```

Comment:
This is testing the specific change to the parsing function, but isn't testing anything related to the original issue. It's not at all clear from the change and the test what actually happens now in the original case of the issue this PR is intended to fix. Does returning null instead of throwing work, or does it just create a different issue elsewhere in that codepath?

If the goal is to fix the case of an SVG with percentage values or `none` values, we should have a test that exercises that, and ensures that the higher-level behavior is as desired with this change.

Reply:
hi @stuartmorgan , what kind of test cases need to be added, do you have any suggestions?
currently I am also trying to read the documentation

Reply:
You provided an SVG that triggered this error in a comment above; a test that parsed a minimal version of that SVG and ensured that it behaved as expected seems like it would give good coverage of this use case.